testing/ci/e2e_ci_prepare_env.sh

```diff
@@ -7,6 +7,9 @@ if [[ ! -d ~/go ]]; then
     wget -q -O - https://raw.githubusercontent.com/canha/golang-tools-install-script/master/goinstall.sh | bash
 fi
 
+pip3 install nearup --user
+
 export GOROOT=~/.go
 export GOPATH=~/go
 export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+export PATH=$HOME/.local/bin:$PATH
```

Comment:
this should be added here, as the problem will persist for normal users, explained above